setup.sh

```diff
@@ -446,6 +447,32 @@ function install_home()
   remove_legacy_git_from_kernel_config_manager
   # Update version based on the current branch
   update_version
+  # Show current environment in terminal
+  setup_bashrc_to_show_current_kw_env
+}
+
+function setup_bashrc_to_show_current_kw_env()
+{
+  local config_files_path="$etcdir"
+  local config_file_template="${config_files_path}/git_prompt_current_env_name.sh"
+  local shellrc="${HOME}/.bashrc"
+  local content_current_env_file
+
+  if [[ $(ask_yN 'Do you want to customize the bash terminal to show the current env?') =~ '1' ]]; then
```

Comment:
If the user selects `--setup-env-notification` it means they want this feature. In other words, it does not make sense to ask this question. You may wish to alert that the bashrc will be updated or something like that.